### DISCUSSION TITLE
Running Cron Job in Vite/Nitro Server

### DISCUSSION THREAD

**Discussion by nvijayap** [vueform/vueform#89] [Questions] [answered]
Is it possible to run a cron job in the `Vite/Nitro server`?  If yes, how?

**Comment by nvijayap** [comment, marked as answer]
May not be relevant for this repo

**Reply by adamberecz** [reply]
Yes, this might not be the best place to get an actual answer. It's better to visit Vite's repo:
https://github.com/vitejs/vite
